Traceback (most recent call last):
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 150, in spec
    return self.env_specs[id]
KeyError: 'ALE/Pong-v4'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/elliot/Documents/GitHub/PhD/Modern Implementations/DQN_etc/DQN_Classes.py", line 256, in <module>
    env = gym.make(config["env_name"])
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 184, in make
    return registry.make(id, **kwargs)
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 105, in make
    spec = self.spec(path)
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 161, in spec
    raise error.DeprecatedEnv(
gym.error.DeprecatedEnv: Env ALE/Pong-v4 not found (valid versions include ['Pong-v0', 'Pong-v4'])
Traceback (most recent call last):
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 150, in spec
    return self.env_specs[id]
KeyError: 'ALE/Pong-v4'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/elliot/Documents/GitHub/PhD/Modern Implementations/DQN_etc/DQN_Classes.py", line 256, in <module>
    env = gym.make(config["env_name"])
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 184, in make
    return registry.make(id, **kwargs)
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 105, in make
    spec = self.spec(path)
  File "/home/elliot/.local/lib/python3.8/site-packages/gym/envs/registration.py", line 161, in spec
    raise error.DeprecatedEnv(
gym.error.DeprecatedEnv: Env ALE/Pong-v4 not found (valid versions include ['Pong-v0', 'Pong-v4'])